test/unified_format.py

```diff
@@ -226,7 +226,10 @@ def add_event(self, event):
         if event_name in self._event_types:
             super(EventListenerUtil, self).add_event(event)
         for id in self._event_mapping[event_name]:
-            self.entity_map[id].append(str(event))
+            self.entity_map[id].append({
+                "name": type(event).__name__,
```

Comment:
`repr(event)` is better than just the type name because it includes the event name and other additional info (see the `__repr__` methods in monitoring.py).

Reply:
I originally had it just use str(event). However,
```
 if ('name' not in event) or (not event['name'].endswith('Event')):
                    self.fail("The workload executor didn't record event name as expected.")
```
This line from validator.py in drivers-atlas-testing seems to indicate that it expects it to just have the name, and no additional information

Reply:
Okay, in that case can we add a new field for the full event repr?

Reply:
Will do.